doc/source/parameters/dem/simulation_control.rst

```diff
@@ -0,0 +1,30 @@
+Simulation Control
+-------------------
+This subsection contains the general information of the simulation, including ``time step``, ``end time``, ``output path``, ``log frequency``, and ``output frequency``. It is the most commonly modified section for a simulation. ``time step`` in DEM simulations is generally in the range of 1e-7 to 1e-5 seconds. With this small ``time step``, the DEM simulation can capture a single collision in several iterations, which leads to the high accuracy of the simulation. 
+
+.. note::
+    Lethe-DEM compares the selected ``time step`` to Rayleigh time-step and prints a warning if the selected ``time step`` is too small or too large. Users should modify the ``time step`` if they see this warning.
```

Comment:
It would be good to add the formula to the Rayleigh time step to which it is being compared to.

Reply:
Added.